react-projects/amysong21/my-app/src/index.js

```diff
@@ -0,0 +1,68 @@
+import ReactDOM from 'react-dom';
+import React from 'react';
+import './index.css';
+
+class Square extends React.Component {
+    render() {
+      return (
+        <button className="square">
+          {/* TODO */}
```

Comment:
Looks like you still have some parts of the tutorial to finish. Let us know in Slack if you need help!